integration/SysSite.spec.ts

```diff
@@ -0,0 +1,744 @@
+        .send({ repositoryName: mockRepositoryName2 })
+
+      // Assert
+      expect(actual.statusCode).toBe(expected)
+    })
+
+    it("should return 400 when createdBy is not an email address", async () => {
+      // Arrange
+      const expected = 400
+
+      // Act
+      const actual = await request(app)
+        .patch(`/${mockRepositoryName}`)
+        .send({ createdBy: "Arthur" })
+
+      // Assert
+      expect(actual.statusCode).toBe(expected)
+    })
+
+    it("should return 400 when launchedBy is not an email address", async () => {
+      // Arrange
+      const expected = 400
+
+      // Act
+      const actual = await request(app)
+        .patch(`/${mockRepositoryName}`)
+        .send({ launchedBy: "Arthur" })
+
+      // Assert
+      expect(actual.statusCode).toBe(expected)
+    })
+
+    it("should return 400 when repositoryUrl is not a URI", async () => {
+      // Arrange
+      const expected = 400
+
+      // Act
+      const actual = await request(app)
+        .patch(`/${mockRepositoryName}`)
+        .send({ repositoryUrl: "Hitchhiker's Guide" })
+
+      // Assert
+      expect(actual.statusCode).toBe(expected)
+    })
+
+    it("should return 400 when stagingUrl is not a URI", async () => {
+      // Arrange
+      const expected = 400
+
+      // Act
+      const actual = await request(app)
+        .patch(`/${mockRepositoryName}`)
+        .send({ stagingUrl: "Hitchhiker's Guide" })
+
+      // Assert
+      expect(actual.statusCode).toBe(expected)
+    })
+
+    it("should return 400 when productionUrl is not a URI", async () => {
+      // Arrange
+      const expected = 400
+
+      // Act
+      const actual = await request(app)
+        .patch(`/${mockRepositoryName}`)
+        .send({ productionUrl: "Hitchhiker's Guide" })
+
+      // Assert
+      expect(actual.statusCode).toBe(expected)
+    })
+
+    it("should return 400 when uptimeUrl is not a URI", async () => {
+      // Arrange
+      const expected = 400
+
+      // Act
+      const actual = await request(app)
+        .patch(`/${mockRepositoryName}`)
+        .send({ uptimeUrl: "Hitchhiker's Guide" })
+
+      // Assert
+      expect(actual.statusCode).toBe(expected)
+    })
+
+    it("should return 400 when launchedAt is not an ISO Date String", async () => {
+      // Arrange
+      const expected = 400
+
+      // Act
+      const actual = await request(app)
+        .patch(`/${mockRepositoryName}`)
+        .send({ launchedAt: "The end of the universe" })
+
+      // Assert
+      expect(actual.statusCode).toBe(expected)
+    })
+
+    it("should return 400 when redirectFrom is not an array of strings", async () => {
+      // Arrange
+      const expected = 400
+
+      // Act
+      const actual = await request(app)
+        .patch(`/${mockRepositoryName}`)
+        .send({ redirectFrom: [42] })
+
+      // Assert
+      expect(actual.statusCode).toBe(expected)
+    })
+  })
+
+  // describe("/email/verifyOtp", () => {
```

Comment:
we could use `describe.only` for this behaviour (see [here](https://jestjs.io/docs/api#describeonlyname-fn))